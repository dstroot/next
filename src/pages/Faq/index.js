import React, { useEffect } from 'react';

// component
import SectionHero from '../../components/Sections/SectionHero';
import Footer from '../../components/Footer';

const Faq = () => {
  // Set the page title and position using the useEffect hook
  useEffect(() => {
    document.title = `NEXT â€¢ FAQ`;
    window.scrollTo(0, 0);
  });

  return (
    <div>
      <SectionHero
        heading={
          <div>
            <h1 className="nxt_heading-xlarge">FAQ</h1>
          </div>
        }
        content={<div />}
      />
      <Footer />
    </div>
  );
};

export default Faq;
