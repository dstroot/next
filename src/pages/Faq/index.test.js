import React from 'react';
import { create } from 'react-test-renderer';
import Faq from '../Faq';

describe('Faq', () => {
  it('it should render', () => {
    let tree = create(<Faq />);
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
