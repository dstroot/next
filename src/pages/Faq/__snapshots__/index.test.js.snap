// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Faq it should render 1`] = `
<div>
  <div
    className="bg-light py-6"
  >
    <div
      className="nxt_heading-xlarge text-center"
    >
      FAQ
    </div>
    <div
      className="my-5 text-center"
    >
      All references to “you or your” assumes the insured and policy owner are the same person.
    </div>
    <div
      className="container"
    >
      <div
        className="row mb-4"
      >
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Speech Bubble"
              className="nxt_icon pb-3"
              src="speech_bubble.svg"
            />
            <div>
              Frequently used terms
            </div>
          </div>
        </div>
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Plant"
              className="nxt_icon pb-3"
              src="plant.svg"
            />
            <div>
              Life insurance basics
            </div>
          </div>
        </div>
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Shield"
              className="nxt_icon pb-3"
              src="shield.svg"
            />
            <div>
              Next Term Life
            </div>
          </div>
        </div>
      </div>
      <div
        className="row mb-4"
      >
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Checkbox"
              className="nxt_icon pb-3"
              src="checkbox.svg"
            />
            <div>
              Eligibility
            </div>
          </div>
        </div>
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Clipboard"
              className="nxt_icon pb-3"
              src="clipboard.svg"
            />
            <div>
              Applying
            </div>
          </div>
        </div>
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Moneybag"
              className="nxt_icon pb-3"
              src="moneybag.svg"
            />
            <div>
              Premium Payment
            </div>
          </div>
        </div>
      </div>
      <div
        className="row mb-4"
      >
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Legal Document"
              className="nxt_icon pb-3"
              src="legal_document.svg"
            />
            <div>
              Managing Your Policy
            </div>
          </div>
        </div>
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="Dollar Document"
              className="nxt_icon pb-3"
              src="dollar_document.svg"
            />
            <div>
              Claims
            </div>
          </div>
        </div>
        <div
          className="col-sm text-center"
        >
          <div
            className="card h-40 py-4"
          >
            <img
              alt="People Group"
              className="nxt_icon pb-3"
              src="people_group.svg"
            />
            <div>
              Beneficiaries
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="container my-5"
  >
    <div
      className="nxt_heading-large"
    >
      Create the accordian here
    </div>
  </div>
  <div
    className="container-fluid bg-dark"
  >
    <div
      className="row"
    >
      <div
        className="col p-3"
      >
        <img
          alt="Logo"
          className="pr-3 pb-4"
          src="NXT_logo_footer_dark.svg"
        />
        <span
          className="nxt_footer"
        >
          <p>
            By using this website you accept our
             
            <a
              href="/terms"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Terms of Use
              </span>
            </a>
            ,
             
            <a
              href="/social"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Social Media Guidelines
              </span>
            </a>
            , and
             
            <a
              href="/privacy"
              onClick={[Function]}
            >
              <span
                className="text-accent"
              >
                Privacy Policy
              </span>
            </a>
          </p>
        </span>
        <small>
          <div
            className="nxt_footer"
          >
            <p>
              No bank guarantee • Not a deposit • May lose value • Not FDIC/NCUA insured • Not insured by any federal government agency
            </p>
            <p>
              Pacific Life, its affiliates, its distributors, and respective representatives do not provide tax, accounting, or legal advice. Any taxpayer should seek advice based on the taxpayer's particular circumstances from an independent tax advisor or attorney.
            </p>
            <p>
              <i>
                Pacific Life is a product provider. It is not a fiduciary and therefore does not give advice or make recommendations regarding insurance or investment products.
              </i>
            </p>
            <p>
              Pacific Life Insurance Company (Newport Beach, CA) is licensed to issue insurance products in all states except New York. Product availability and features may vary by state. Fixed annuity products are available through licensed third parties.
            </p>
            <p>
              © 
              2019
               Pacific Life Insurance Company, Newport Beach, CA.
            </p>
          </div>
        </small>
      </div>
    </div>
  </div>
</div>
`;
